src/Core/Utilities/JsonHelpers.cs

```diff
@@ -45,18 +45,6 @@ static JsonHelpers()
         };
     }
 
-    [Obsolete("This is built into .NET 6, it SHOULD be removed when we upgrade")]
-    public static T ToObject<T>(this JsonElement element, JsonSerializerOptions options = null)
-    {
-        return JsonSerializer.Deserialize<T>(element.GetRawText(), options ?? Default);
-    }
-
-    [Obsolete("This is built into .NET 6, it SHOULD be removed when we upgrade")]
-    public static T ToObject<T>(this JsonDocument document, JsonSerializerOptions options = null)
-    {
-        return JsonSerializer.Deserialize<T>(document.RootElement.GetRawText(), options ?? default);
-    }
```

Comment:
Oh thank you!

Reply:
This was an easy one. The others referenced by the ".NET 6" comments are expansive enough in testing to me to debate doing now.